commands/commands.go

```diff
@@ -99,10 +99,11 @@ type nodeDefinition struct {
 }
 
 type clusterDefinition struct {
-	Name           string           `yaml:"name,omitempty"`
-	Owner          string           `yaml:"owner,omitempty"`
-	ReleaseVersion string           `yaml:"release_version,omitempty"`
-	Workers        []nodeDefinition `yaml:"workers,omitempty"`
+	AvailabilityZones int              `yaml:"availability_zones,omitempty"`
+	Name              string           `yaml:"name,omitempty"`
+	Owner             string           `yaml:"owner,omitempty"`
+	ReleaseVersion    string           `yaml:"release_version,omitempty"`
+	Workers           []nodeDefinition `yaml:"workers,omitempty"`
 }
```

Comment:
As this is used to create YAML, I'd appreciate moving `AvailabilityZones` below `ReleaseVersion`. And yes, there is no simpe rule for this.

Reply:
Done.